src/ol_dbt/models/marts/combined/marts__combined__users.sql

```diff
@@ -0,0 +1,134 @@
+--- This model combines intermediate users from different platforms
+
+with mitx__users as (
+    select * from {{ ref('int__mitx__users') }}
+)
+
+, mitxpro_users as (
+    select * from {{ ref('int__mitxpro__users') }}
+)
+
+, bootcamps_users as (
+    select * from {{ ref('int__bootcamps__users') }}
+)
+
+, combined__users as (
+    select
+        user_mitxonline_id
+        , user_edxorg_id
+        , null as user_mitxpro_id
+        , null as user_bootcamps_id
+        , user_mitxonline_username
+        , user_edxorg_username
+        , null as user_mitxpro_username
+        , null as user_bootcamps_username
+        , case
+            when is_mitxonline_user = false
+                then user_edxorg_email
+            when is_edxorg_user = false
+                then user_mitxonline_email
+            when user_joined_on_mitxonline > user_joined_on_edxorg
+                then user_mitxonline_email
+            else coalesce(user_edxorg_email, user_mitxonline_email)
+        end as user_email
```

Comment:
If I read it correctly, this can be written to:
```suggestion
        , case
            when user_joined_on_mitxonline > user_joined_on_edxorg
                then user_mitxonline_email
            else coalesce(user_edxorg_email, user_mitxonline_email)
        end as user_email
```

Reply:
I changed it for simplicity thanks